docs/docs/docs/quick-start/_category_.yml

```
@@ -0,0 +1,4 @@
+position: 1
+label: '🚀 Quick Start'
+collapsible: true
+collapsed: false 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)